app/res/layout/hierarchy_layout.xml

```diff
@@ -40,40 +40,31 @@ the License.
             android:layout_height="match_parent"
             android:layout_weight="1"/>
 
-        <Button
+        <com.google.android.material.button.MaterialButton
             android:id="@+id/jumpPreviousButton"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:text="@string/jump_to_previous"
             android:padding="12dp"
             android:textSize="@dimen/text_medium"
-            android:textColor="@color/cc_brand_color"
-            android:background="@color/cc_core_bg"
-            style="@style/CommCare.Button.Default"
             android:layout_weight="1"/>
 
-        <Button
+        <com.google.android.material.button.MaterialButton
             android:id="@+id/jumpBeginningButton"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:text="@string/jump_to_beginning"
             android:padding="12dp"
             android:textSize="@dimen/text_medium"
-            android:textColor="@color/cc_brand_color"
-            android:background="@color/cc_core_bg"
-            style="@style/CommCare.Button.Default"
             android:layout_weight="1"/>
 
-        <Button
+        <com.google.android.material.button.MaterialButton
             android:id="@+id/jumpEndButton"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:text="@string/jump_to_end"
             android:padding="12dp"
             android:textSize="@dimen/text_medium"
-            android:textColor="@color/cc_brand_color"
-            android:background="@color/cc_core_bg"
-            style="@style/CommCare.Button.Default"
             android:layout_weight="1"/>
```

Comment:
I think that visually this looks fine but there are a few aspects to address:
(1) when disabled, the contrast between the background and the text makes the label unreadable
(2) there is space between the left edge and the first button, making it look misaligned
(3) maybe we should capitalize all the letters as it is now
![image](https://github.com/user-attachments/assets/a1ae84c7-a7d7-4b4d-b59e-87eb9a34bc00)
this is how it looks now:
![image](https://github.com/user-attachments/assets/4158feea-12e4-42b7-bcd3-5984f6d966f5)